components/engine_traits/src/engine.rs

```diff
@@ -63,5 +66,120 @@ pub trait KvEngine:
 // It should be named as `EngineFactory` for consistency, but we are about to rename
 // engine to tablet, so always use tablet for new traits/types.
 pub trait TabletFactory<EK> {
```

Comment:
Would it be better to use a `TabletFactoryV2` instead? I understand it contains disjoint sets of interfaces. (e.g. old raftstore only use `create_shared_db` and should have access to other methods).

Reply:
No, old raftstore will likely use the new APIs in some code to share with V2.